luma/core/legacy/font.py

```diff
@@ -8,6 +8,9 @@
 +-------------------+-------------------------------------------------+------------------------------------------------------------------------------------------------+
 | Font name         | Notes                                           | Source                                                                                         |
 +===================+=================================================+================================================================================================+
+| ``ATARI_FONT``    | Designed for a terminal emulator written for    | http://gnu.ethz.ch/linuks.mine.nu/atari/                                                       |
```

Comment:
this link doesn't resolve. But better to have something that nothing..?

Reply:
yep, lets leave the link in just in case it ever came back